### DISCUSSION TITLE
Reverse Proxy - jc21/nginx-proxy-manager

### DISCUSSION THREAD

**Discussion by martinorpa** [nextcloud/all-in-one#605] [Questions] [answered]
My server is running many containers, so it has **jc21/nginx-proxy-manager**  as a Reverse Proxy.

We set admin.ourdomain.com pointing to https://Server_IP_address:8080.

When NextCloud ask for domain, we set to nextcloud.ourdomain.com and we point this domain to  http://Server_IP_address:11000

Containers are running except Apache and Nextcloud. Both show **Starting** .

Nextcloud log's shows: _psql: error: connection to server at "nextcloud-aio-database" (172.22.0.5), port 5432 failed: FATAL:  password authentication failed for user "oc_nextcloud" Configuring Redis as session handler_

Apache log's shows: _nc: getaddrinfo for host "nextcloud-aio-nextcloud" port 9000: Name or service not known
Waiting for Nextcloud to start..._

Do you guys know a possible solution?

Thank You !!

**Comment by szaimen** [comment, marked as answer]
Hello, can you please try to purge the instance and start from the beginning?

Or is there already data present on this instance?

This looks like a hickup to me and should usually not happen.

**Comment by martinorpa** [comment]
For now I quit (don't have more time), part of the roadmap would be a easier reverse-proxy configuration.

on the server containers are using  image: jwilder/nginx-proxy and image: jrcs/letsencrypt-nginx-proxy-companion

Will try another day

Thank you !!!
